src/jsp/jsp_sr.c

```diff
@@ -290,13 +322,37 @@ delay_load_hook (unsigned dliNotify, PDelayLoadInfo pdli)
 
 	if (java_home)
 	  {
+	    err_msgs.append ("\n\tFailed to load libjvm from 'JAVA_HOME' environment variable: ");
```

Comment:
is it right that error message append to err_msgs, if java_home is not null ?

Reply:
The same reason with above